packages/jss/src/utils/escape.js

```diff
@@ -1,15 +1,10 @@
-const env = process.env.NODE_ENV
-
-const escapeRegex = /([[\].#*$><+~=|^:(),"'`])/g
-
 export default str => {
   // We don't need to escape it in production, because we are not using user's
```

Comment:
On Material-UI, we still escape in production as people can use a global CSS approach, where they provide a name to each style sheet.

Reply:
It's what we use at https://www.onepixel.com/en/ to get some decent caching performance.

Reply:
If they do this, they also use valid strings for class names, why do you still need escaping there?

Reply:
yes, react-jss does too, exactly the same story